python/ray/dashboard/modules/job/common.py

```diff
@@ -66,6 +66,26 @@ def is_terminal(self) -> bool:
         return self.value in {"STOPPED", "SUCCEEDED", "FAILED"}
 
 
+@PublicAPI(stability="stable")
+class JobErrorType(str, Enum):
+    """An enumeration for describing the error type of a job."""
+
+    # Runtime environment failed to be set up
+    RUNTIME_ENV_SETUP_FAILURE = "RUNTIME_ENV_SETUP_FAILURE"
+    # Job supervisor actor launched, but job failed to start within timeout
+    JOB_START_TIMEOUT = "JOB_START_TIMEOUT"
+    # Job supervisor actor failed to start
+    JOB_SUPERVISOR_ACTOR_START_FAILURE = "JOB_SUPERVISOR_ACTOR_START_FAILURE"
+    # Job supervisor actor failed to be scheduled
+    JOB_SUPERVISOR_ACTOR_UNSCHEDULABLE = "JOB_SUPERVISOR_ACTOR_UNSCHEDULABLE"
+    # Job supervisor actor failed for unknown exception
+    JOB_SUPERVISOR_ACTOR_UNKNOWN_FAILURE = "JOB_SUPERVISOR_ACTOR_UNKNOWN_FAILURE"
+    # Job driver script failed to start due to exception
+    JOB_DRIVER_SCRIPT_START_ERROR = "JOB_DRIVER_SCRIPT_START_ERROR"
+    # Job driver script failed due to non-zero exit code
+    JOB_DRIVER_SCRIPT_ERROR = "JOB_DRIVER_SCRIPT_ERROR"
```

Comment:
Should we use DRIVER_SCRIPT or ENTRYPOINT_COMMAND

Seems we are mix using driver and entrypoint command

Reply:
IMO should be `entrypoint` because that's the public API "entrypoint: python main.py"